promoter/file/filestore.go

```diff
@@ -73,11 +91,7 @@ func openFilestore(
 	}
 
 	if u.Scheme != "gs" {
-		return nil, fmt.Errorf(
-			"unrecognized scheme %q (supported schemes: %s)",
-			filestore.Base,
-			object.GcsPrefix,
-		)
+		return nil, fmt.Errorf("unrecognized scheme %q, expected gs://", filestore.Base)
```

Comment:
Same comment about `object.GcsPrefix` as above.

Reply:
Done - put these into the api, as I think they are _our_ values, not the library's values (i.e. if the library changed the value, we would not want to change our values!)